tests/orchestrator/setEnvVars.ts

```diff
@@ -0,0 +1,5 @@
+export function setEnv(envConfig: { [key: string]: string }) {
+  Object.keys(envConfig).forEach((key) => {
+    process.env[key] = envConfig[key];
+  });
+}
```

Comment:
Is this being called somewhere? What is invoking this? I don't see it.

Reply:
FWIW, I'm not asking because I'm worried about about dead code.  I'm asking, because a key part of this is how we're managing the sets of environment variables in the orchestrator.